smart_contracts/contracts/explorer/faucet-stored/src/main.rs

```diff
@@ -28,32 +42,74 @@ fn store() -> (ContractHash, ContractVersion) {
     let entry_points = {
         let mut entry_points = EntryPoints::new();
 
-        let entry_point = EntryPoint::new(
-            ENTRY_POINT_NAME,
+        let faucet = EntryPoint::new(
+            ENTRY_POINT_FAUCET,
+            vec![Parameter::new(
+                ARG_ID,
```

Comment:
I think we should make the relevant consts public in the lib so we can use them here, e.g.
```suggestion
                faucet::ARG_ID,
```